pkg/sysinfo/cloud/cloud_test.go

```diff
@@ -28,31 +28,31 @@ func (s *CloudDetectionSuite) TestParseAWSMeta(c *C) {
 	mux.Handle("/valid", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Content-Type", "text/plain")
 		w.WriteHeader(http.StatusOK)
-		w.Write([]byte("i-db519dd1\n"))
+		_, _ = w.Write([]byte("i-db519dd1\n"))
```

Comment:
Why do we need to do this assignment?

Reply:
Just making the linter/IDE happy here as it complains about not handling the error here. As this is just a test I'm not interested in validating if there is an error so just ignoring it.